### PR TITLE
Enable more ADC1 channels (pins 32-39)

### PR DESCRIPTION
**Pull request by Idorobots**
### Description

Enabled more ADC1 channels in the HAL initialization routines.

### Benefits

More temperature sensors can now be hooked up to ESP32-based targets.

### Related Issues

None.

### CODE REVIEW

File: Marlin/src/HAL/HAL_ESP32/HAL.cpp [hunk 1]
```diff
@@ -147,8 +151,15 @@ adc1_channel_t get_channel(int pin) {
...
   adc1_config_width(ADC_WIDTH_12Bit);
-  adc1_config_channel_atten(get_channel(36), ADC_ATTEN_11db);
   adc1_config_channel_atten(get_channel(39), ADC_ATTEN_11db);
+  adc1_config_channel_atten(get_channel(36), ADC_ATTEN_11db);
+  adc1_config_channel_atten(get_channel(35), ADC_ATTEN_11db);
+  adc1_config_channel_atten(get_channel(34), ADC_ATTEN_11db);
+  adc1_config_channel_atten(get_channel(33), ADC_ATTEN_11db);
+  adc1_config_channel_atten(get_channel(32), ADC_ATTEN_11db);
+
+  // Note that adc2 is shared with the WiFi module, which has higher priority, so the conversion may fail.
```
Comment (Idorobots) [inline Marlin/src/HAL/HAL_ESP32/HAL.cpp:161]: There still might be some utility in enabling ADC2 channels, but I think we're good with 6 channels of ADC1 for now.

**Comment by thinkyhead** [comment]
Commit messages should follow the documented form and use the proper tense:

- **BAD:** "Enabled more ADC1 channels (pins 32-39)."
- **GOOD:** "Enable more ADC1 channels (pins 32-39)."

**Comment by thinkyhead** [comment]
Please remove this text when posting PRs:
```
### Requirements

* Filling out this template is required. Pull Requests without a clear description may be closed at the maintainers' discretion.
```

**Comment by Idorobots** [comment]
Fixed commit messages.

**Pull request merged**